.github/scripts/generateSpecMapping.js

```diff
@@ -0,0 +1,55 @@
+import fs from 'node:fs';
+import path from 'node:path';
+import { fileURLToPath } from 'node:url';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+// Add new specs to be deployed to Bump here
+const SPEC_MAPPING = [
+  {
+    doc: 'f929d2d7-27d7-4591-b22f-6c2e543a7874',
+    file: 'openapi/v1-deprecated/v1.json',
+    branch: 'main',
+  },
+];
+
+function handleAdminAPIv2() {
+  const docId = '2accf4b8-a543-426c-94c3-794ae26b68be';
```

Comment:
can we pass this ID as env variable? you can add another secret or var to the repo